neqo-http3/src/connection_client.rs

```diff
@@ -291,7 +293,16 @@ impl Http3Client {
 
     pub fn process_output(&mut self, now: Instant) -> Output {
         qtrace!([self], "Process output.");
-        self.conn.process_output(now)
+
+        // Maybe send() stuff on http3-managed streams
+        self.process_http3(now);
```

Comment:
If I look at the standard approach here, it seems like process_http3 is called after each of the transport processing functions.  So maybe you can skip this one.

Reply:
no we cannot skip this one. this repeating of process_http3 is why I have not done i earlier, I wanted to look if we can do this better, just I didn't have time.
I am also a bit unhappy about reading events, I think it is fragile because I am afraid that we will have a bug that some old evens are kept that should not be kept (we already had one of those which was a panic in neqo code), but I have not had time to look into that.

Reply:
we can land this and I will look at process_http3 if I find time